<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>538</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>35</pointsize>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color : #002223;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="boared_widget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>57</y>
      <width>481</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="game_bar">
     <item row="2" column="2">
      <widget class="QPushButton" name="btn_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 32px;
font-weight: bold;
background-color: transparent;
border: 2px solid #ffffff;
border-top: 2px solid black;
border-left: 2px solid black;
border-right: none;
border-bottom: none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRepeatDelay">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btn_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 32px;
font-weight: bold;
background-color: transparent;
border: 2px solid #ffffff;
border-top: none;
border-left: none;
border-right: 2px solid black;
border-bottom: 2px solid black;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRepeatDelay">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 32px;
font-weight: bold;
background-color: transparent;
border: 2px solid #ffffff;
border-top: 2px solid black;
border-left: 2px solid black;
border-right: none;
border-bottom: 2px solid black;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRepeatDelay">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="btn_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 32px;
font-weight: bold;
background-color: transparent;
border: 2px solid #ffffff;
border-top: none;
border-left: 2px solid black;
border-right: 2px solid black;
border-bottom: 2px solid black;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRepeatDelay">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 32px;
font-weight: bold;
background-color: transparent;
border: 2px solid #ffffff;
border-top: 2px solid black;
border-left: 2px solid black;
border-right: 2px solid black;
border-bottom: 2px solid black;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRepeatDelay">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="btn_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 32px;
font-weight: bold;
background-color: transparent;
border: 2px solid #ffffff;
border-top: 2px solid black;
border-left: none;
border-right: 2px solid black;
border-bottom: none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRepeatDelay">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 32px;
font-weight: bold;
background-color: transparent;
border: 2px solid #ffffff;
border-top: 2px solid black;
border-left: none;
border-right: 2px solid black;
border-bottom: 2px solid black;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRepeatDelay">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btn_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 32px;
font-weight: bold;
background-color: transparent;
border: 2px solid #ffffff;
border-top: none;
border-left: 2px solid black;
border-right: none;
border-bottom: 2px solid black;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRepeatDelay">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="btn_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <pointsize>-1</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font-size: 32px;
font-weight: bold;
background-color: transparent;
border: 2px solid #ffffff;
border-top: 2px solid black;
border-left: 2px solid black;
border-right: 2px solid black;
border-bottom: none;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="autoRepeatDelay">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="sidebar_widget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>481</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="top_bar">
     <property name="spacing">
      <number>24</number>
     </property>
     <property name="leftMargin">
      <number>25</number>
     </property>
     <property name="rightMargin">
      <number>245</number>
     </property>
     <item>
      <widget class="QPushButton" name="about">
       <property name="styleSheet">
        <string notr="true">color:white;
border-bottom: none;</string>
       </property>
       <property name="text">
        <string>about</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="new_game">
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>new game</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>425</y>
      <width>481</width>
      <height>61</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QHBoxLayout" name="score_bar">
     <property name="spacing">
      <number>30</number>
     </property>
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <item>
      <widget class="QLineEdit" name="player1_score">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
background-color : #40E0D0;</string>
       </property>
       <property name="text">
        <string>     You(X) : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cpu">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;
background-color:orange;</string>
       </property>
       <property name="text">
        <string>   Player2(O) : </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="tie">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white;</string>
       </property>
       <property name="text">
        <string>    Ties :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>370</y>
      <width>125</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="game_mode_bar">
     <item>
      <widget class="QRadioButton" name="tow_player">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;</string>
       </property>
       <property name="text">
        <string>play with player</string>
       </property>
       <property name="iconSize">
        <size>
         <width>27</width>
         <height>25</height>
        </size>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="vs_cpu">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">color:white;
</string>
       </property>
       <property name="text">
        <string>play with cpu</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>508</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
